### PR TITLE
Write readme

### PR DESCRIPTION
**Pull request by potsbo**
## WHY

[Rich readme · Issue #37 · wantedly/developers-account-mapper](https://github.com/wantedly/developers-account-mapper/issues/37)

## WHAT

- [x] Summary
- [x] Description
- [x] Usage
  - [x] AWS configuration
  - [x] Command usage
  - [x] docker
- [x] Install
- [x] Contribution
- [x] Author
- [x] License

### CODE REVIEW

**Comment by potsbo** [comment]
https://github.com/wantedly/developers-account-mapper/blob/potsbo/doc/README.md

**Comment by potsbo** [comment]
@dtan4 Could you review this please

File: README.md [hunk 1]
```diff
@@ -3,11 +3,77 @@
...
+$ developers-account-mapper to-slack-mention potsbo
+<@U2XXXXXXX|shimpei>
+```
 ## Description
 
+Convert a key username, which is recommended to be unique among the developers group, to other username.
+Currently, we support Slack and GitHub.
+
+### Slack
+Especially abount Slack, developers-account-mapper is able to fetch Slack ID by Slack username and output it as a Slack mention format.
```
Comment (dtan4) [inline README.md]: `abount` -> `about`

**Review by dtan4 — CHANGES_REQUESTED** [review]

**Comment by dtan4** [comment]
Please add empty line {before,after} {headers,codeblocks}.

File: README.md (hunk 1 above)
Comment (potsbo) [reply README.md]: Fixed, thank you

**Comment by potsbo** [comment]
Empty line added.
It's ready.

File: README.md [hunk 2]
```diff
@@ -3,11 +3,81 @@
...
 
+One DynamoDB tables named `DevelopersAccountMap` has to be created.
+
+#### `DevelopersAccountMap` table
+
+|Key|Type| |
+|---|----|---|
+|LoginName|String|Primary key|
+
+In terraform, you can create the table with below.
```
Comment (dtan4) [inline README.md]: `Terraform` is the correct spell.
https://www.terraform.io/

Comment (potsbo) [reply README.md]: Thanks
And it should be a link.

**Comment by potsbo** [comment]
@dtan4 Fixed

**Review by dtan4 — APPROVED** [review]

**Comment by dtan4** [comment]
LGTM 👍

**Pull request merged**